CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ While some features like the clipboard, menus or file dialogs are not yet availa
 - Timer events will only be delivered to the widgets that requested them. ([#831] by [@sjoshid])
 - `Event::Wheel` now contains a `MouseEvent` structure. ([#895] by [@teddemunnik])
 - `AppDelegate::command` now receives a `Target` instead of a `&Target`. ([#909] by [@xStrom])
+- `SHOW_WINDOW` and `OPEN_WINDOW` no longer require a `WindowId` as payload, but they must `Target` the correct window. ([#???] by [@finnerale])
```

Comment:
maybe this can just be a separate patch?

Reply:
I suppose this could be split into
- Updating `SHOW_WINDOW` and `OPEN_WINDOW` to use `Target`
- Fix open and save menu items
- Make `Selector` type safe
- Split `Selector` into `Selector` and `SelectorOnce`
- Add type-safe? methods to `EventCtx` to create menus and windows.

Splitting the last two out would cause quite some duplicated / wasted work tho, as `SelectorOnce` influences error and return types, while menus creation decides over adding wrapper types or new methods.

Reply:
Even the first one means that I'll have to first declare it to take `WidgetId` and than remove that in a separate PR again.

Reply:
yea, if things are truly entwined than doing it together makes sense, but if there are good dividing points that does make review a lot easier.